### PR TITLE
Notification Bars for Update flow

### PR DESCRIPTION
**Pull request by blis-mendix**

### CODE REVIEW

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/merge-algorithm.md [hunk 1]
```diff
@@ -269,3 +271,71 @@ Remember that Mine and Theirs are different, depending on whether you are using
...
+
+## 5 Notification Controls {#notifications}
+
+While combining changes either by rebase or merge you will be shown a notification bar which will inform about current status of the process and next available steps.
+
+Some of the controls are shared between both processes
+
+### 5.1 Shared
+
+#### 5.1.1 'Abort merge'
```
Comment (tknippenberg) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/merge-algorithm.md]: 5.1.1 'Abort' 
Instead of 'Abort merge'?

Comment (blis-mendix) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/merge-algorithm.md]: Yep, done!

**Review by MarkvanMents — APPROVED** [review]
@blis-mendix 
I've proofread this and slightly reorganized the examples. Looks good.
@MariaShaposhnikova, do you have any suggestions?
I've also taken the opportunity to move all the images into the merge-algorithm folder for consistency.
I'm approving this, as I am happy for it to be merged for 10.6.

**Comment by blis-mendix** [comment]
@MarkvanMents thanks for the update! Everything looks great, nice idea with cutting the images will look hell of a better!

**Review by MariaShaposhnikova — APPROVED** [review]
@MarkvanMents , thank you for your review! I made very small changes and I am happy to merge this PR for 10.6

**Comment by blis-mendix** [comment]
Hi @MariaShaposhnikova , @MarkvanMents  not sure if @tknippenberg mention but this is not for 10.6 but for 10.6.1 currently because we missed the 10.6 freeze with feature.

**Comment by MarkvanMents** [comment]
Thanks, I've updated the label so it doesn't get merged accidentally.

**Comment by Adam-Dupaski** [comment]
@MarkvanMents @MariaShaposhnikova publishing for 10.6.1 release

**Comment by MarkvanMents** [comment]
@Adam-Dupaski - thanks for picking this up.

**Pull request merged**